wasm/binary/import.go

```diff
@@ -69,3 +69,12 @@ func encodeImport(i *wasm.Import) []byte {
 	}
 	return data
 }
+
+// encodeCode returns the wasm.Code encoded in WebAssembly 1.0 (MVP) Binary Format.
+//
+// See https://www.w3.org/TR/wasm-core-1/#binary-code
+func encodeCode(c *wasm.Code) []byte {
+	code := append(leb128.EncodeUint32(c.NumLocals), c.LocalTypes...)
```

Comment:
I think this encoding maybe isn't correct as 

local types are encoded as `uint32 (number of locals for specific type), byte (value type),...` 

See https://github.com/tetratelabs/wazero/blob/main/wasm/binary/code.go#L31-L47